include/boost/geometry/strategy/spherical/envelope_range.hpp

```diff
@@ -165,9 +165,6 @@ inline bool pole_within(T const& lat_pole, Ring const& ring,
     }
 
     using point_t = typename geometry::point_type<Ring>::type;
-    using coord_t = typename geometry::coordinate_type<point_t>::type;
-    using units_t = typename geometry::detail::cs_angular_units<point_t>::type;
-    using constants_t = math::detail::constants_on_spheroid<coord_t, units_t>;
```

Comment:
This is unrelated, but it gave me warnings in many files I compiled.
So I thought it's better to get this in too on short term.